test/boilerplate_suggestors/advanced_props_and_state_class_migrator_test.dart

```diff
@@ -1492,16 +2187,155 @@ void main() {
                 }
               }
             ''',
-            );
+          );
 
-            expect(converter.convertedClassNames, {
-              'ADifferentPropsClass': 'ADifferentPropsClassMixin',
-              'FooPropsMixin': 'FooPropsMixin',
-              'FooProps': 'FooProps',
-            });
-          });
+          expect(
+              converter.convertedClassNames,
+              isValidFilePath
+                  ? {
+                      'ADifferentPropsClass': 'ADifferentPropsClassMixin',
+                      'FooPropsMixin': 'FooPropsMixin',
+                      'FooProps': 'FooProps',
+                    }
+                  : {
+                      'ADifferentPropsClass': 'ADifferentPropsClassMixin',
+                      'FooPropsMixin': 'FooPropsMixin',
+                    });
         });
       });
     });
   });
 }
+
+exportLocationsCommentTestHelper({
```

Comment:
This just needs a return type
```suggestion
void exportLocationsCommentTestHelper({
```